corehq/toggles.py

```diff
@@ -884,7 +884,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
 INDEX_LOCATION_DATA = StaticToggle(
     'index_location_data',
     'Add option to index custom location fields',
-    TAG_EXPERIMENTAL,
+    TAG_SOLUTIONS,
```

Comment:
@sheelio I'm not sure `SOLUTIONS` is the right tag for this feature. `INTERNAL` maybe?

Reply:
I think CUSTOM for now (since its for enikshay).